osu.Game/Rulesets/Mods/ModRateAdjust.cs

```diff
@@ -27,5 +28,9 @@ public virtual void ApplyToSample(IAdjustableAudioComponent sample)
         public override string SettingDescription => SpeedChange.IsDefault ? string.Empty : $"{SpeedChange.Value:N2}x";
 
         public override string ExtendedIconInformation => SettingDescription;
+
+        public override bool Ranked =>
+            // only allow speed change to be ranked to be sure that any mods that inherit this one don't accidentally make new settings ranked too
+            SettingsBindables.Where(s => s != SpeedChange).All(s => s.IsDefault);
```

Comment:
Will this make Adjust Pitch switch unranked?

Reply:
Yes, as written in the comment above the code. While it's only a "cosmetic" setting I think it's better to only allow speed adjusting for now anyway

Reply:
The Adjust Pitch switch is ranked right now, and there's no reason to unrank it